Automattic/jetpack
# E2E tests: mention users in Slack based on report name and branch name

#### Changes proposed in this Pull Request:

This adds more flexibility and control over user mentions in Slack notifications. We can now configure lists of Slack handles to be mentioned for a branch or for a report name (e.g. `atomic` or `gutenberg`)

#### Jetpack product discussion
1156386383419466-as-1200924739850035

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- CI green
- It's a bit harder to test locally. You'll need to set an env variable GITHUB_CONTEXT with the GitHub event information you want to test (pull_request, push, scheduled). You can get the content of such event by searching in actions history. Then, you'll need the summary.json file in the `output` folder. For this, you can run the tests locally. Then run the script with `NODE_ENV=test node ./bin/slack.js suite suiteName`. See script's help for more options.
